app/views/layouts/application.html.erb

```diff
@@ -35,8 +35,9 @@
           <%= flash[:alert] %>
         </div>
       <% end %>
-
-      <%= yield %>
+      <div style="max-width: 1280px; margin: 0 auto; padding: 1rem;">
+        <%= yield %>
+      </div>
```

Comment:
I think websites look a little more polished when the content doesn't go all the way to the edige of the screen on larger monitors.